packages/ui/README.md

```diff
@@ -110,3 +110,39 @@ By default the application uses base configuration from `path_to_dist/server/con
 
 - [`YT_AUTH_CLUSTER_ID`](https://github.com/ytsaurus/ytsaurus-ui/blob/ui-v1.16.1/packages/ui/README.md#environment-variables) environment variable has been replaced by [`ALLOW_PASSWORD_AUTH`](https://github.com/ytsaurus/ytsaurus-ui/blob/main/packages/ui/README.md#environment-variables).
 - [`config.ytAuthCluster`](https://github.com/ytsaurus/ytsaurus-ui/blob/ui-v1.16.1/packages/ui/src/%40types/core.d.ts#L75) option has been replaced by [`config.allowPasswordAuth`](https://github.com/ytsaurus/ytsaurus-ui/blob/ui-v1.17.0/packages/ui/src/%40types/core.d.ts#L16).
+
+
+### How to run e2e on local machine
+
+Here is an example of run & update of screenshot tests:
+```bash
+# Prerequsites: install docker and YTsaurus CLI
+# 
+# https://ytsaurus.tech/docs/en/api/cli/install
+
+# Terminal 1: launch local dev mode
+npm run dev:localmode:e2e
+
+# Terminal 2: init local cluster
+npm run e2e:localmode:init
+
+# Terminal 2: mount repo in docker image and prepare everyting for tests run
+docker run --rm --network host -it -w /work \
+    -v $(pwd):/work \
+    "ubuntu:jammy" \
+    /bin/bash -c '
+            apt-get update
+            apt-get install curl -y
+            curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash 
+            export NVM_DIR="$HOME/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
+            nvm install v18.19.0
```

Comment:
All the lines above might be removed if we use base image from our [Dockerfile](https://github.com/ytsaurus/ytsaurus-ui/blob/main/packages/ui/Dockerfile#L1), isn't it?